etc/kayobe/apt.yml

```diff
@@ -78,6 +78,22 @@ stackhpc_apt_repositories:
     signed_by: ceph.asc
     architecture: amd64
     required: true
+  - url: "{{ stackhpc_repo_ubuntu_noble_url }}"
```

Comment:
These (and the jammy ones) need to be conditional, otherwise all ubuntu hosts will get repos for both jammy and Noble